src/components/NftGallery/NftGallery.tsx

```diff
@@ -59,7 +59,7 @@ function NftGallery({ collection, mobileLayout }: Props) {
 const StyledCollectionNfts = styled.div<{ columns: number; mobileLayout: DisplayLayout }>`
   display: grid;
   grid-template-columns: ${({ columns, mobileLayout }) =>
-    mobileLayout === DisplayLayout.LIST ? '1fr' : `repeat(${columns},  minmax(auto, 50%))`};
+    mobileLayout === DisplayLayout.LIST ? '1fr' : `repeat(${columns},  minmax(auto, 100%))`};
```

Comment:
always have the child fit to 100% of its space; the child's container will determine how wide it gets based on # of columns